### DISCUSSION TITLE
How can I get a githubstar?

### DISCUSSION THREAD

**Discussion by iNuts-Web** [community/community#54271] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

A few days ago I saw a "GithubStar" badge on a friend's account, how do I get one?

**Comment by lucasscaramelo** [comment, marked as answer]
The seal talks about people who help the development community in general, you can get it by helping others in your area, with videos, ebooks, lectures or something that brings value to the community and that people can recognize your value in that, see more here: https://stars.github.com/

**Comment by iNuts-Web** [comment]
Cool! Thanks a lot!
